etc/ci/upload_docs.sh

```diff
@@ -16,5 +16,7 @@ cp etc/doc.servo.org/* target/doc/
 
 python components/style/properties/build.py servo html
 
+OUT_DIR=`pwd`/target/doc make -f makefile.cargo -C components/script dom_docs
```

Comment:
Please use `OUT_DIR="$(pwd)/target/doc"` per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).